### PR TITLE
Add sake and soy sauce recipe

### PR DESCRIPTION
**Pull request by AtomicFox556**
#### Summary
Content "Add sake and soy sauce recipe"

#### Purpose of change

Adding more craftable foodstuffs.
#### Describe the solution

Add sake, a Japanese alcoholic beverage made from rice and recipe to make it.
Add recipes to produce soy sauce from wheat flour and soy beans.
Add recipe to produce wild koji mold for sake production in case one can't just find it.
Fermentation times are shortened like they appear to be for the rest of fermented stuff, and recipes are a bit simplified to require less individual recipes as crafting steps.
#### Describe alternatives you've considered

None.
#### Testing

Made both sake and soy sauce in-game, noticed no errors.
#### Additional context

Partially taken from Youkai Mod.

### CODE REVIEW

File: data/json/items/comestibles/brewing.json [hunk 1]
```diff
@@ -141,6 +141,73 @@
...
+    "type": "COMESTIBLE",
+    "id": "soy_sauce_unstrained",
+    "name": { "str_sp": "unstrained soy sauce" },
+    "description": "Nearly finished soy sauce that still has some solid particles floating in it.  Still needs to be strained, and also pasteurized to get rid of harmful bacteria.",
+    "weight": "24 g",
+    "color": "white",
+    "container": "bottle_glass",
+    "symbol": "~",
+    "quench": -20,
+    "healthy": -1,
```
Comment (NetSysFire) [inline data/json/items/comestibles/brewing.json:199]: If there are harmful bacteria in it, I would make the health penalty a bit bigger and maybe even give it a chance to induce vomiting.

File: data/json/recipes/food/other.json [hunk 2]
```diff
@@ -50,6 +50,52 @@
...
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_OTHER",
+    "skill_used": "cooking",
+    "difficulty": 5,
+    "time": "4 h",
+    "proficiencies": [ { "proficiency": "prof_food_prep" }, { "proficiency": "prof_fermenting" } ],
+    "book_learn": [ [ "cookbook_sushi", 6 ] ],
+    "qualities": [ { "id": "COOK", "level": 3 } ],
+    "tools": [ [ [ "surface_heat", 80, "LIST" ] ], [ [ "mortar_pestle", -1 ], [ "food_processor", 20 ] ] ],
+    "components": [ [ [ "soybean", 1 ] ], [ [ "flour", 10 ] ], [ [ "water", 4 ], [ "water_clean", 4 ] ], [ [ "paper", 1 ] ] ]
```
Comment (NetSysFire) [inline data/json/recipes/food/other.json:80]: Would it be possible to be able to use another wheat-free flour here?

**Review by NetSysFire — COMMENTED** [review]
Those changes look pretty detailed to me. Well done so far.

File: data/json/recipes/food/other.json (hunk 2 above)
Comment (AtomicFox556) [reply data/json/recipes/food/other.json:80]: Not really; most soy sauces specifically are made with wheat flour in addition to soybeans. Tamari soy sauce is made with just soybeans and there are some gluten-free soy sauce substitutions made in a completely different way, but no recipes seem to use non-wheat flour.

**Pull request merged**